packages/cli/src/zosfiles/create/ds/ds.definition.ts

```diff
@@ -58,7 +58,8 @@ export const DsDefinition: ICommandDefinition = {
     examples: [
         {
             description: strings.ACTIONS.DATA_SET_LIKE.EXAMPLES.EX1,
-            options: "NEW.DATASET --like EXISTING.DATASET"
+            options: {"NEW.DATASET --like EXISTING.DATASET",
+                      "NEW.DATASET --like EXISTING.DATASET --lrecl 1024"
```

Comment:
The `options` property doesn't support an array with multiple values. Please add the second example as a new `{description: string, options: string}` object in the `examples` array.

Reply:
After making this change, please update unit and integration test snapshots to reflect the updated command help. 🙂